data/Badbatunde.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Habeeb B. Oyesile",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Tech, Open-source, Finance and Books",
+  "avatar": "https://github.com/Badbatunde/png",
```

Comment:
```suggestion
```

Reply:
The change has been made as requested.

The update docs link you attached displayed a '404: This page cannot be found' error by the way.